decidim-admin/config/locales/ca.yml

```diff
@@ -636,4 +636,4 @@ ca:
   time:
     formats:
       datepicker: "%m/%d/%Y"
-      timepicker: "%d/%m/%Y %H:%M"
\ No newline at end of file
+      timepicker: "%d/%m/%Y %H:%M"
```

Comment:
Also, they said:

> According to [YAML specification](http://www.yaml.org/spec/1.2/spec.html), all strings that contain special characters (?, ¿, [], {} etc.) should be wrapped with the quotes. 

But as far as I can understand, it doesn't prefer one type of quotes over the other one 😕